packages/process/CHANGELOG.md

```diff
@@ -1,5 +1,6 @@
-## NEXT
+## 5.0.3
```

Comment:
The changelog and pubspec changes for this package can be reverted, since now you're only adding a non-doc comment to an implementation file, which doesn't affect clients.